docs/examples/api.rst

```diff
@@ -0,0 +1,19 @@
+API Example Client
+===============
+
+This simple client is designed to showcase the API by demonstrating
+the correct sequence of function calls used in an elecition.
+
+.. warning::
+
+   This example client **does not** do any sort of error handling in
```

Comment:
We do need an example code that does error handling. I am OK with having the "simple" example without error handling to show the steps; but we definitely need to show how error handling must be done with this API.

Reply:
made #75